queens-psyc-clinic/app
# Fix backups

Creates a volume (`db/data`) of the db data so that if the database goes down, it will restart with the most recent data.

Also changes `backup_job.py` to copy `db/data/psychClinic` so that if a catastrophic failure happens is production, the data is saved in `db/backup` and can be restored

### Testing

1. Deploy the app (using `docker-compose.delpoy.yml`)
2. Add new data (ie. a new user)
3. Stop the app and use `docker compose -f docker-compose.delpoy.yml down` (normally this should wipe the database)
4. restart the app (like in 1.) and check if the data is still there
